### PR TITLE
add article & aside part

### PR DESCRIPTION
**Pull request by Boam1994**
Hey girls, i add article & aside part but i have some difficulties with css. can you have a look, correct if you want or can and merge it to master.
Thanks

### CODE REVIEW

**Review by murvanessa — CHANGES_REQUESTED** [review]
you forgot to change dev strat

File: index.html [hunk 1]
```diff
@@ -14,12 +14,30 @@
...
+        <h3>Section</h3>
+      </section>    
+      <article>
+        <h1>An amazing article about a collaborative project</h1>
+        <p>Hey all! Look we created our first integrate project using branches and pull requests. It's really exciting  to work together.<br>
+              Nevertheless, it's not something easy. We have to understand each other and make some conventions about how to name branches, what everyone have to do; so on and so on...
+        </p>
+        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nemo, unde et. Soluta, mollitia odit sint accusantium iste molestiae sequi, nemo dolores magni totam repellendus quisquam quas id libero laudantium hic?</p>
+      </article>
+    </div>
```
Comment (Feruzteame) [inline index.html:32]: can you remove this div. and can you remove the p. there is no need text.

**Review by Feruzteame — CHANGES_REQUESTED** [review]

**Review by murvanessa — COMMENTED** [review]
good

File: index.html (hunk 1 above)
Comment (Boam1994) [reply index.html:32]: I think it changes nothing to the exercice

**Comment by snicoll** [comment]
Great collaboration here, nicely done!

**Pull request merged**